content/en/tracing/setup/nodejs.md

```diff
@@ -82,7 +82,20 @@ const tracer = require('dd-trace').init({
 
 ## Compatibility
 
-Node `^4.7`, `^6.9` and `>=8` are supported by this library. However, it benefits significantly from the performance improvements introduced in Node `>=8.3`.
+Node `>=4` is supported by this library. We recommend always using the latest LTS to avoid known issues and performance degradation. Odd version numbers are not officially supported.
```

Comment:
```suggestion
Only Node `>=4` is supported by this library but you should use the latest LTS to avoid known issues and performance degradation. 
```